test/common/http/mixed_conn_pool_test.cc

```diff
@@ -106,11 +106,19 @@ void MixedConnPoolImplTest::testAlpnHandshake(absl::optional<Protocol> protocol)
   conn_pool_.reset();
 }
 
-TEST_F(MixedConnPoolImplTest, BasicNoAlpnHandshake) {
+TEST_F(MixedConnPoolImplTest, BasicNoAlpnHandshakeOld) {
```

Comment:
Do we need an h3 pool test too?

Reply:
I don't think so - there's no changes to the H3 pool